Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:bookworm-slim
+
+WORKDIR /abacus
+
+# Copy the binary, database and migrations
+COPY --chown=abacus:abacus ./backend/target/release/api /abacus/backend
+COPY --chown=abacus:abacus ./backend/db.sqlite /abacus/db.sqlite
+COPY --chown=abacus:abacus ./backend/migrations /abacus/migrations
```

Comment:
You'll need to create this user+group and also run using this user, right?

Something like:
```Dockerfile
RUN groupadd -r abacus --gid=999 && useradd --no-log-init -r -g abacus --uid=999 abacus
USER 999
```

Reply:
Fixed in [5bcf348](https://github.com/kiesraad/abacus/pull/429/commits/5bcf348fc5def4b7c545d1e5234625595c110e93)